aries-site/src/components/feedback/Question.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormField,
+  TextArea,
+  Text,
+  RadioButtonGroup,
+  FileInput,
+} from 'grommet';
+import { StarRating } from './StarRating';
+import { ThumbsRating } from './ThumbsRating';
+
+export const Question = ({ formProps, label, inputProps, kind, name }) => {
+  const accessibilityProps = {
+    id: name,
+    name,
+  };
+
+  const formats = {
```

Comment:
```suggestion
  const kinds = {
```

Reply:
can this variable be pulled out of the Question and just be standalone?

Reply:
im not sure I can try

Reply:
^^ bumping this again so the object is outside of the component.